hbase-common/src/main/java/org/apache/hadoop/hbase/nio/HBaseReferenceCounted.java

```diff
@@ -40,7 +40,7 @@ default boolean release(int increment) {
 
   @Override
   default HBaseReferenceCounted touch() {
-    throw new UnsupportedOperationException();
+    return touch(null);
```

Comment:
this change just makes it 1 step easier to enable this for other subclasses, since they could just override `touch(Object hint)` below and get both. It's functionally equivalent to today, since both throw an exception at present.